src/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine.php

```diff
@@ -84,20 +77,11 @@ public function __construct(
      */
     public function execute()
     {
-        $searchConfig = $this->searchEngineConfig->get();
+        $config = $this->searchEngineConfig->getConfig();
```

Comment:
Better to move these under try-catch, because methods `getConfig` or `getName` can throw an exception even such is not described in methods docs.

Reply:
Fixed